src/components/dst-legend.tsx

```diff
@@ -30,7 +46,7 @@ const tileSelection: ITileSelection = {
 // should probably follow the same pattern if so.
 const dataDisplayLayout = new DataDisplayLayout({
   tileWidth: 500,
-  tileHeight: 470,
+  tileHeight: 440,
```

Comment:
This is changed again in the next PR. I might backport that change to this PR so things aren't broken if there is a long pause between this PR and the next one.